### ISSUE TITLE
Wish I could write Author

### ISSUE THREAD

**Issue by casedup** [Apress/learn-php-8#1]
Trying to read through this book but its so poorly organized it's hard, very hard. Why does the author write incomplete code only to complete it later on?? This book reads like spaghetti. If the author is reading can you PLEASE for the love of god try to write a book that finishes the code and complete a thought all in one clean swoop. 

Stop bouncing all over the place, might have to put this one down as the code is so hard to get through because it is incomplete then picks up later on

**Comment by kosjir** [comment]
Hi casedup, I totally agree with you. It is very hard to follow the text in book and the source code. Some files are missing, wrong references etc. I have already got to Chapter 5 and have succeeded to "repair" the source, but have not documented these repaires, but if I can be of any help, pls let me know.
